mlir/python/CMakeLists.txt

```diff
@@ -311,6 +311,14 @@ declare_mlir_dialect_python_bindings(
     dialects/rocdl.py
   DIALECT_NAME rocdl)
 
+declare_mlir_python_sources(
+  MLIRPythonSources.Dialects.transform.extras
+  ADD_TO_PARENT MLIRPythonSources.Dialects
+  ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/mlir"
+  GEN_ENUM_BINDINGS
+  SOURCES
+    dialects/transform/extras/__init__.py)
```

Comment:
umm can we keep all extras under one `extras`? one of these should be suitable:

1. `dialects/extras/transform/__init__.py`
2. `extras/dialects/transform/__init__.py`

The second is preferrable because moving `extras/types.py` to ~ `dialects/extras/types.py` doesn't really make sense.

Reply:
Thank you for the suggestion, I went with 2. 👍